sdk/compiler/daml-lf-proto-util/src/DA/Daml/LF/Proto3/Util.hs

```diff
@@ -20,3 +25,76 @@ encodeHash = T.pack . reverse . foldl' toHex [] . BS.unpack
         [n1, n2] -> n2 : n1 : xs
         [n2]     -> n2 : '0' : xs
         _        -> error "impossible: showHex returned [] on Word8"
+
+
+------------------------------------------------------------------------
+-- Shorthands
+------------------------------------------------------------------------
+
+-- Kinds
+liftK :: P.KindSum -> P.Kind
+liftK = P.Kind . Just
+
+pkstar :: P.Kind
+pkstar = (liftK . P.KindSumStar) P.Unit
+
+pknat :: P.Kind
+pknat = (liftK . P.KindSumNat) P.Unit
+
+pkarr :: P.Kind -> P.Kind -> P.Kind
+pkarr k1 k2 = liftK $ P.KindSumArrow $ P.Kind_Arrow (V.singleton k1) (Just k2)
+
+pkinterned :: Int32 -> P.Kind
+pkinterned = liftK . P.KindSumInterned
+
+-- Types
+pliftT :: P.TypeSum -> P.Type
+pliftT = P.Type . Just
+
+pbuiltin :: P.BuiltinType -> P.Type
+pbuiltin bit = pliftT $ P.TypeSumBuiltin $ P.Type_Builtin (P.Enumerated $ Right bit) V.empty
+
+ptunit :: P.Type
+ptunit = pbuiltin P.BuiltinTypeUNIT
+
+ptint :: P.Type
+ptint = pbuiltin P.BuiltinTypeINT64
+
+ptbool :: P.Type
+ptbool = pbuiltin P.BuiltinTypeBOOL
```

Comment:
A lot of these could also be pattern synonyms, but I think it's a lot less urgent, so completely up to you if you want that in this PR.